### PR TITLE
Day one

### PR DESCRIPTION
**Pull request by Talaspaev**
My first PR.I have watched 13 videos from 100.First day has come to an end.
image

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (Dmitry-White) [inline src/App/App.js]: Consider separating third-party imports from the project's ones.
So first imports in a file should be third-party imports like "react".
Then your project common imports that appear in almost all the components - like for example "../../utils/your-util.js".
Then your component related stuff - like "./your-component-helper.js".
And then the css imports.

Every import group should be separated from other import groups with an empty new line.

File: src/App/App.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import './App.css';
+import Header from './../Components/Header/header'
```
Comment (Dmitry-White) [inline src/App/App.js]: Consider removing "./" part from the import string - it is redundant.

File: src/Components/Header/header.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import logo from "./2222.jpg";
```
Comment (Dmitry-White) [inline src/Components/Header/header.jsx]: Consider following the import comment above.

File: src/Components/Navbar/nav.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+const NavBar = () => {
```
Comment (Dmitry-White) [inline src/Components/Navbar/nav.jsx]: Consider separating it with an empty new line.

File: src/Components/Profile/Profile.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import logo from "./2222.jpg";
```
Comment (Dmitry-White) [inline src/Components/Profile/Profile.jsx]: Consider following the import comments above.

File: src/App/App.js (hunk 1 above)
Comment (Talaspaev) [reply src/App/App.js]: I've fixed my jsx files.Now it looks nice.
image

File: src/App/App.js (hunk 2 above)
Comment (Talaspaev) [reply src/App/App.js]: i've removed the excess "./" part
image

File: src/Components/Header/header.jsx (hunk 3 above)
Comment (Talaspaev) [reply src/Components/Header/header.jsx]: i've added new line between imports
image

File: src/Components/Navbar/nav.jsx (hunk 4 above)
Comment (Talaspaev) [reply src/Components/Navbar/nav.jsx]: I've added new line between import and function
image

File: src/Components/Profile/Profile.jsx (hunk 5 above)
Comment (Talaspaev) [reply src/Components/Profile/Profile.jsx]: i've added new line between imports
image

**Pull request merged**